acp/acp.go

```diff
@@ -13,23 +13,18 @@ package acp
 import (
 	"context"
 
-	"github.com/sourcenetwork/corelog"
+	protoTypes "github.com/cosmos/gogoproto/types"
+
 	"github.com/sourcenetwork/immutable"
 
 	"github.com/sourcenetwork/defradb/acp/identity"
+	acpTypes "github.com/sourcenetwork/defradb/acp/types"
 )
 
-var (
-	log = corelog.NewLogger("acp")
-
-	// NoACP is an empty ACP, this is used to disable access control.
-	NoACP = immutable.None[ACP]()
-)
-
-// ACP is the interface to all types of access control that might exist.
-type ACP interface {
+// ACPSystemClient is an abstraction to allow multiple types of ACP systems to share DefraDB specific logic.
+type ACPSystemClient interface {
```

Comment:
Note: diff looks odd but it's just copy pasted of what sourcehub client was before (+ some renames) and this `ACP` interface is now `DocumentACP` under `acp/dac`. Both being used just like they were before.